package.json

```diff
@@ -43,6 +43,7 @@
 		"compare-versions": "^6.1.0",
 		"dayjs": "^1.11.9",
 		"discord-rpc": "^4.0.1",
+		"pnpm": "^8.10.0",
```

Comment:
i think you mean to run `npm i -g pnpm` instead of adding it to the repo? there is no need for crankshaft to require pnpm
```suggestion

```

Reply:
I did run that command afterwards, I had a bit of trial-and-error because I hadn't ever used pnpm before. I'll remove this in my next PR.

Reply:
you can just commit new commits to your fork and it will show up in this pr as long as i have not merged it yet